opta/core/kubernetes.py

```diff
@@ -37,10 +37,6 @@
 
 import opta.constants as constants
 from opta.constants import GENERATED_KUBE_CONFIG_DIR, REDS, yaml
-from opta.core.aws import AWS
-from opta.core.azure import Azure
-from opta.core.gcp import GCP
-from opta.core.local import Local
```

Comment:
it's nice that k8s is no longer aware of each cloud impl

Reply:
yup!